### PR TITLE
Fix CI error

### PR DESCRIPTION
**Pull request by Sija**
Blocked by https://github.com/luckyframework/lucky_flow/pull/68

### CODE REVIEW

**Comment by paulcsmith** [comment]
Looks like the LuckyFlow shard needs to be bumped to 0.6.1 https://github.com/crystal-loot/exception_page/blob/4c013ce8fe2a39c0e2f47b4c7a8fa459ad4a1943/shard.yml#L11

**Comment by paulcsmith** [comment]
Looks like a chromedriver issue. We may need to get new binaries and add them to the vendor folder

**Comment by Sija** [comment]
I managed to install an up-to-date version of the ChromeDriver, there are still some issues left though...

File: spec/spec_helper.cr [hunk 1]
```diff
@@ -1,15 +1,14 @@
...
+require "./support/*"
 
 include LuckyFlow::Expectations
 
-server = TestServer.new(3002)
+server = TestServer.new
 
 LuckyFlow.configure do |settings|
-  settings.base_uri = "http://localhost:3002"
+  settings.base_uri = "http://#{server.addr}"
```
Comment (paulcsmith) [inline spec/spec_helper.cr:11]: Nice 👍

**Comment by paulcsmith** [comment]
Thanks for doing this 👍 We've finally got it green!

**Pull request merged**